packages/editor-ui/src/views/SettingsCommunityNodesView.vue

```diff
@@ -10,18 +10,24 @@
 					@click="openNPMPage"
 				/>
 			</div>
-			<div :class="$style.loadingContainer" v-if="isLoading">
-				<n8n-spinner size="large"/>
-				<n8n-text>{{ $locale.baseText('settings.communityNodes.loading.message') }}</n8n-text>
```

Comment:
does this need to be removed from en.json?

Reply:
Yup.